lib/mocha.js

```diff
@@ -1219,22 +1221,40 @@ Mocha.prototype.runGlobalTeardown = async function runGlobalTeardown(
 };
 
 /**
- * Run global fixtures sequentially with context `context`
+ * Run global fixtures sequentially with context `context`.
  * @private
  * @param {MochaGlobalFixture[]} [fixtureFns] - Fixtures to run
  * @param {object} [context] - context object
  * @returns {Promise<object>} context object
  */
 Mocha.prototype._runGlobalFixtures = async function _runGlobalFixtures(
   fixtureFns = [],
-  context = {}
+  context = {},
 ) {
-  for await (const fixtureFn of fixtureFns) {
-    await fixtureFn.call(context);
+  for (const fixtureFn of fixtureFns) {
+    try {
+      await fixtureFn.call(context);
+    } catch (err) {
+      if(context.stats) {
```

Comment:
[Question] When would `context.stats` not exist?

I'm wondering if the parameters to the `_runGlobalFixtures` function can be assumed to exist. I.e. that the `= []` and `= {}` are unnecessary. I don't see any way in code to have them not exist.